libs/components/indicators/src/lib/modules/status-indicator/status-indicator.component.ts

```diff
@@ -85,7 +85,9 @@ export class SkyStatusIndicatorComponent {
   constructor(
     private changeDetector: ChangeDetectorRef,
     private resources: SkyLibResourcesService
-  ) {}
+  ) {
+    this.updateIcon();
```

Comment:
This seems wrong to do in the constructor because the input won't be set at this point. It's a side-effect of us erroneously returning a default value in a `get` accessor which is forbidden in our coding conventions.

Reply:
This was just in a weird state as we hadn't strict moded this component and so I didn't make any of those changes here. I went ahead and just made those changes here so that the component follows our conventions. I just hadn't done that as this component hadn't gone through that work yet.

Even with these changes - I still need to call `updateIcon` in the constructor as we still have a default that needs to have this function called for it.

Reply:
We chatted at the end of the week and I believe the outcome was to leave this in the constructor and not put it in `ngOnInit`. Please let me know if I misunderstood the resolution to that discussion.